plotly/utils.py

```diff
@@ -356,3 +356,8 @@ def is_source_key(key):
         return True
     else:
         return False
+
+PLOTLY_OFFLINE_DIRECTORY = plotlyjs_path = os.path.expanduser(
+    os.path.join(*'~/.plotly/plotlyjs'.split('/')))
+PLOTLY_OFFLINE_BUNDLE = os.path.join(PLOTLY_OFFLINE_DIRECTORY,
+                                     'plotly-ipython-offline-bundle.js')
```

Comment:
.. so that `exceptions.py` *and* `offline.py` can use these variables without circular imports